docs/serving/using-cert-manager-on-gcp.md

```diff
@@ -167,45 +165,23 @@ exists.
    where `<your-domain.com>` is your domain:
 
    ```shell
-   # Change this value to the domain you want to use.
-   export DOMAIN=<your-domain.com>
-
-   kubectl apply --filename - <<EOF
-   apiVersion: certmanager.k8s.io/v1alpha1
-   kind: Certificate
-   metadata:
-     name: my-certificate
-     # Istio certs secret lives in the istio-system namespace, and
-     # a cert-manager Certificate is namespace-scoped.
-     namespace: istio-system
-   spec:
-     # Reference to the Istio default cert secret.
-     secretName: istio-ingressgateway-certs
-     acme:
-       config:
-       # Each certificate could rely on different ACME challenge
-       # solver.  In this example we are using one provider for all
-       # the domains.
-       - dns01:
-           provider: cloud-dns-provider
-         domains:
-         # Since certificate wildcards only allow one level, we will
-         # need to one for every namespace that Knative is used in.
-         # We don't need to use wildcard here, fully-qualified domains
-         # will work fine too.
-         - "*.default.$DOMAIN"
-         - "*.other-namespace.$DOMAIN"
-     # The certificate common name, use one from your domains.
-     commonName: "*.default.$DOMAIN"
-     dnsNames:
-     # Provide same list as `domains` section.
-     - "*.default.$DOMAIN"
-     - "*.other-namespace.$DOMAIN"
-     # Reference to the ClusterIssuer we created in the previous step.
-     issuerRef:
-       kind: ClusterIssuer
-       name: letsencrypt-issuer
-   EOF
+  # Change this value to the domain you want to use.
+  export DOMAIN=<your-domain.com>
+
+  kubectl apply --filename - <<EOF
+  apiVersion: cert-manager.io/v1alpha2
+  kind: Certificate
+  metadata:
+    name: my-certificate
+    namespace: istio-system
+  spec:
+    secretName: istio-ingressgateway-certs
```

Comment:
Do these docs need to be istio specific?

Reply:
This doc mainly shows how to auto provision certs from lets encrypt with dns challenge handled by google cloud dns. We are just using istio as an example to show how certs are configured. I guess it could be other gateways as well.